.circleci/config.yml

```diff
@@ -159,6 +159,7 @@ jobs:
             JEST_JUNIT_OUTPUT_DIR: "/tmp/test-results/integration-test-results.xml"
             SCREENSHOT_DIR: "/tmp/failure-screenshots"
           command: |
+            mkdir -p /tmp/failure-screenshots
```

Comment:
Assuming we have access to the env above, this could be `mkdir -p ${SCREENSHOT_DIR}`.